catalog/scripts/gen_type_asserts.py

```
@@ -0,0 +1,81 @@
+import typing as t
+from pathlib import Path
+from textwrap import dedent
+import sys
+
+def get_funcs(models: t.Iterable[Path]) -> t.Iterator[str]:
+    for path in models:
+        with path.open() as f:
+            # skip boilerplate
+            lines = iter(f.readlines())
+            while not next(lines).startswith("package"):
```

Pylint (3.3.7)

[refactor] Do not raise StopIteration in generator, use return statement instead

(R1708)